src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -24,9 +24,9 @@ <h2 class="addon"{{ addon.name|locale_html }} id="addon-name">
       <span class="version">{{ version.version }} ({{ version.get_channel_display() }})</span>
     {% endif %}
   </h2>
-  <h4 class="author">by
-    {% for author in addon.listed_authors %}
-    <a href="{{ url('reviewers.developer_profile', author.id) }}">{{ author.name }}</a>{% if not loop.last %},{% endif %}
+  <h4 class="author">by 
+    {% for author in addon.current_authors %}
+    <a href="{{ url('reviewers.developer_profile', author.id) }}" style="font-weight:{{ 'bold' if author in addon.listed_authors else 'normal' }}">{{ author.name }}</a>{% if not loop.last %},{% endif %}
```

Comment:
Could you avoid inline styles and create a class for this ? We currently allow inline styles in CSP because of existing old code but in general we try to avoid that.